### PR TITLE
Edit page for key deletion as well as expiring.

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

**Comment by letzya** [comment]
@marksou what is the difference? GH doesn't show it. thanks

**Comment by letzya** [comment]
Found it's file name key to token?

**Comment by letzya** [comment]
I'm not sure we should do this change per @buger suggestion to add a footer to explain it

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/physical-key-expiry.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Example: To have keys live in Redis for only 24 hours (and be deleted 24 hours post their creation) set it as follow:
+```{.json}
+"session_lifetime": 86400
+```
+If this is not set, then the default is 0 seconds, which means the key will not be deleted from Redis.
+
+This feature works nicely with JWT or OIDC auth methods since the keys get created in Redis the first time they are in use so you know when it will be removed. Be extra careful in the case of keys created by Tyk (Auth token or JWT with individual secrets) and set a big `session_lifetime` otherwise the user might use the key AFTER it has already been removed from Redis.
+
+### Expiring and deleting tokens at the Global level
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/physical-key-expiry.md:38]: but this still had the word "token"

**Comment by marksou** [comment]
@buger could you have another look at this?

**Pull request merged**